public/js/qrcode.js

```diff
@@ -0,0 +1,18 @@
+function generateQRCode() {
```

Comment:
Consider just using jquery here, we rely on it throughout most of our application anyway